addons/website_sale/models/product_template.py

```diff
@@ -250,7 +250,7 @@ def _get_sales_prices(self, pricelist, fiscal_position):
         currency = pricelist.currency_id or self.env.company.currency_id
         date = fields.Date.context_today(self)
 
-        sales_prices = pricelist._get_products_price(self, 1.0)
+        sales_prices = pricelist.sudo()._get_products_price(self, 1.0)
```

Comment:
Good practice: Always justify with a comment why a `.sudo()` is needed. It allows for more easily understanding it can be removed when a later change renders it useless.

Reply:
(Was removed, was part of the fix for wesbite_sale & temporal. fix will be done in https://github.com/odoo/enterprise/pull/49479 fw)